### ISSUE TITLE
Presentation?

### ISSUE THREAD

**Issue by hen** [rgardler/BigLittleChallenge#3]
Perhaps consider putting more of the presentation into the README.

**Comment by SorraTheOrc** [comment]
Yes, I'll fill out the README at some point when I have time and put relevant parts there.
